numpy/random_intel/mklrand/Python.pxi

```diff
@@ -0,0 +1,58 @@
+# :Author:    Robert Kern
+# :Copyright: 2004, Enthought, Inc.
+# :License:   BSD Style
+
+
+cdef extern from "Python.h":
+    # Not part of the Python API, but we might as well define it here.
+    # Note that the exact type doesn't actually matter for Pyrex.
+    ctypedef int size_t
+
+    # String API
+    char* PyString_AsString(object string)
+    char* PyString_AS_STRING(object string)
+    object PyString_FromString(char* c_string)
+    object PyString_FromStringAndSize(char* c_string, int length)
+
+    # Float API
+    double PyFloat_AsDouble(object ob)
+    long PyInt_AsLong(object ob)
+
+    # Memory API
+    void* PyMem_Malloc(size_t n)
+    void* PyMem_Calloc(size_t s, size_t n)
+    void* PyMem_Realloc(void* buf, size_t n)
+    void PyMem_Free(void* buf)
+
+    void Py_DECREF(object obj)
+    void Py_XDECREF(object obj)
+    void Py_INCREF(object obj)
+    void Py_XINCREF(object obj)
+
+    # CObject API
+# If this is uncommented it needs to be fixed to use PyCapsule
```

Comment:
Also note that `PyCapsule` is available in Python 2.7 and we no longer support earlier versions. In fact, PyCapsule has been used in `numpy/random/mtrand/mtrand.pyx` since 1.11.x was branched.